go.mod

```diff
@@ -154,3 +155,5 @@ require (
 )
 
 replace github.com/kubernetes/cri-api => k8s.io/cri-api v0.23.5-rc.0
+
+replace github.com/aquasecurity/tracee/types => ./types
```

Comment:
Move to a types PR once approved.

Reply:
https://github.com/aquasecurity/tracee/pull/3141

Reply:
I'm doing it. Creating another PR (rebase of this without the types change) as well.